submissions/nkeirukaEnendu/HTML-Forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Page</title>
+</head>
+<body>
+    <header>
+        <h1>NextLine Furnitures</h1>
+    </header>
+   <nav>
+       <div>
+           <ul>
+               <li> <a href="#product">Gallery</a></li>
+               <li> <a href="#">FAQs</a></li>
+               <li> <a href="#contactUs">Contact Us</a></li>
+           </ul>
```

Comment:
You havent included any navigation here. Also, you have an opened PR..it's better you ensure an opened PR gets merged before creating another